### PR TITLE
i tried fixing the ghost fumble console time issue

### PR DESCRIPTION
**Pull request by joshypooh**
## About The Pull Request
please dont kill me its my first time

## Why It's Good For The Game
because ghost shouldn't touch this stuff

## Changelog
fixing the ghost fumble console time issue
:cl: add

add: Added more things
del: Removed old things
Fix: fixed a few things
code: changed some code
refactor: refactored some code
config: changed some config setting

/:cl: fix

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
Follow the PR guidelines or maintainers may close this. And draft the PR if you need time in properly making it.

**Comment by SplinterGP** [comment]
> 
> 
> Follow the PR guidelines or maintainers may close this. And draft the PR if you need time in properly making it.

it's alright its their first PR, they're learning

File: code/game/objects/structures/orbital_cannon.dm [hunk 1]
```diff
@@ -461,15 +462,7 @@
...
-	if(user.skills.getRating("engineer") < SKILL_ENGINEER_ENGI)
-		user.visible_message("<span class='notice'>[user] fumbles around figuring out how to use the console.</span>",
-		"<span class='notice'>You fumble around figuring out how to use the console.</span>")
-		var/fumbling_time = 5 SECONDS * ( SKILL_ENGINEER_ENGI - user.skills.getRating("engineer") )
-		if(!do_after(user, fumbling_time, TRUE, src, BUSY_ICON_UNSKILLED))
-			return
-
-	var/dat
-	if(!GLOB.marine_main_ship?.orbital_cannon)
+if(!GLOB.marine_main_ship?.orbital_cannon)
```
Comment (SplinterGP) [inline code/game/objects/structures/orbital_cannon.dm:465]:
```suggestion
	if(!GLOB.marine_main_ship?.orbital_cannon)
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Comment by SplinterGP** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/pull/6671 merged, feel free to ask wayland on help on another Fix pr or something

**Pull request closed without merging**